src/components/VideoProvider/useBackgroundSettings/useBackgroundSettings.test.tsx

```diff
@@ -75,8 +87,14 @@ describe('The useBackgroundSettings hook ', () => {
     expect(backgroundSettings.type).toEqual('none');
   });
 
-  it('should set the background settings correctly and set the video processor when "image" is selected', () => {
-    // TODO add test after implementing virtual background feature/logic
+  it('should set the background settings correctly and set the video processor when "image" is selected', async () => {
+    await act(async () => {
+      setBackgroundSettings(imgSettings as BackgroundSettings);
+    });
+    backgroundSettings = renderResult.current[0];
+    expect(backgroundSettings.type).toEqual('image');
+    expect(backgroundSettings.index).toEqual(2);
+    expect(mockVideoTrack.addProcessor).toHaveBeenCalled();
```

Comment:
Can we have a test that checks addProcessor was called with the virtualBackground param? I think we need to update line 75 as well. Check addProcessor was called with the blur processor param.